packages/plugin-phone/test/integration/spec/phone.js

```diff
@@ -65,19 +64,38 @@ describe(`plugin-phone`, function() {
     });
 
     describe(`#deregister()`, () => {
-      it(`disconnects from mercury`);
-      it(`unregisters from wdm`);
-      it(`is a noop when not registered`);
+      it(`disconnects from mercury`, () => {
+        const mercuryDisconnectSpy = sinon.spy(spock.spark.mercury, `disconnect`);
+        return spock.spark.phone.deregister()
+          .then(() => assert.calledOnce(mercuryDisconnectSpy))
+          .then(() => assert.isFalse(spock.spark.mercury.connected, `Mercury is not connected`))
+          .then(() => assert.isFalse(spock.spark.phone.connected, `Mercury (proxied through spark.phone) is not conneted`))
+          .then(() => mercuryDisconnectSpy.restore());
+      });
+
+      it(`unregisters from wdm`, () =>
+        spock.spark.phone.deregister()
+          .then(() =>
+            spock.spark.device.on(`change:device`)
+              .then(() => assert.isUndefined(spock.spark.device.url))
+          )
+      );
+
+      // TODO: Does not currently noop after multiple calls
+      it.skip(`is a noop when not registered`, () => {
```

Comment:
mercury.disconnect() is the noop in question. maybe spy on spark.request (which is currently impossible, iirc) instead and assert that no network calls get made?